src/main/java/spotify/api/enums/HttpStatusCode.java

```diff
@@ -1,5 +1,15 @@
 package spotify.api.enums;
 
+/**
+ * This class contains and sets the HTTP Status codes when using POST and GET HTTP requests.
```

Comment:
`This class contains all HTTP codes used within the API` is better.

"when using POST and GET HTTP requests" is not true, the API also makes `PUT` and `DELETE` HTTP requests. So I would either fix this or remove it entirely.